README.md

```diff
@@ -91,27 +94,75 @@ Run `npm test` or `yarn test` in the console.
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+Please note we have a [code of conduct](./CODE_OF_CONDUCT.md). Please follow it in all your interactions with the project.
+
+## Issue/Bug Request
+
+**If you are having an issue with the existing project code, please submit a bug report under the following guidelines:**
+
+- Check first to see if your issue has already been reported.
+- Check to see if the issue has recently been fixed by attempting to reproduce the issue using the latest master branch in the repository.
+- Create a live example of the problem.
+- Submit a detailed bug report including your environment & browser, steps to reproduce the issue, actual and expected outcomes, where you believe the issue is originating from, and any potential solutions you have considered.
+
+### Feature Requests
+
+We would love to hear from you about new features which would improve this app and further the aims of our project. Please provide as much detail and information as possible to show us why you think your new feature should be implemented.
+
+### Pull Requests
+
+If you have developed a patch, bug fix, or new feature that would improve this app, please submit a pull request. It is best to communicate your ideas with the developers first before investing a great deal of time into a pull request to ensure that it will mesh smoothly with the project.
+
+Remember that this project is licensed under the MIT license, and by submitting a pull request, you agree that your work will be, too.
+
+#### Pull Request Guidelines
+
+- Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+- Update the README.md with details of changes to the interface, including new plist variables, exposed ports, useful file locations and container parameters.
+- Ensure that your code conforms to our existing code conventions and test coverage.
+- Include the relevant issue number, if applicable.
+- You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
 
-This is version 0.0.3 (Release canvas 2)
 
 ## Authors
 
-* **Pascal Ulor** - *Team lead* - [github](https://github.com/PascalUlor)
-* **Mariam Adedeji** - *Team member* - [github](https://github.com/mariehposa)
-* **Mike Perry Attara** - *Team member* - [github](https://github.com/mikeattara)
-* **Justinas Vebra** - *Team member* - [github](https://github.com/vebradev)
-* **Mildred Otieno** - *Team member* - [github](https://github.com/awuorm)
-* **Ayomide Jones** - *Team member* - [github](https://github.com/Ayormeday)
+| [Anthony Campbell](https://anthonycampbell.dev) | [AbdelIdir ](https://github.com/AbdelIdir)      | [Ekanem David](https://github.com/dueka)        |                                    
```

Comment:
Could you please add `Team Lead - ......` in front of my name to denote clear role distinction?

Reply:
After discussing with others within the group, this has been implemented by adding an additional row stating all posts. This has now been completed.

Reply:
After a discussion with the group, we have added in another row with identifies everyone's role. 
Commpleted